packages/extension-polkagate/src/util/workers/getAssetOnAssetHub.js

```
@@ -1,14 +1,69 @@
+          genesisHash: api.genesisHash.toString(),
+          isAsset: true,
+          isForeignAssets: !!isForeignAssets,
+          priceId: asset?.priceId,
+          token,
+          totalBalance: _balance
+        };
+
+        const _index = addresses[index];
+
+        // @ts-ignore
+        results[_index]?.push(item) ?? (results[_index] = [item]);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)